bosh-director/lib/bosh/director/jobs/db_job.rb

```diff
@@ -31,8 +31,13 @@ def perform
       end
 
       def queue_name
-        @job_class.instance_variable_get(:@queue) ||
-            (@job_class.respond_to?(:queue) and @job_class.queue)
+        if (@job_class.instance_variable_get(:@local_fs) ||
+            (@job_class.respond_to?(:local_fs) and @job_class.local_fs)) && !Config.director_pool.nil?
```

Comment:
May we use `&&` instead of `and`?
There is a difference: http://stackoverflow.com/questions/1426826/difference-between-and-and-in-ruby